internal/backend/db/dbgorm/events.go

```diff
@@ -88,7 +89,7 @@ func (db *gormdb) SaveEvent(ctx context.Context, event sdktypes.Event) error {
 	if connectionID.IsValid() { // only if exists
 		conn, err := db.GetConnection(ctx, connectionID)
 		if err != nil {
-			return err
+			return fmt.Errorf("connection: %w", err)
```

Comment:
better to have ```invalid connection: ``` instead of just ```connection```

Reply:
it's a wrapped error. it will be
"save event: connection: not found"